tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -233,7 +234,7 @@ def scalars_route(self, request):
     # TODO: return HTTP status code for malformed requests
     tag = request.args.get('tag')
     run = request.args.get('run')
-    experiment = request.args.get('experiment')
+    experiment = request.args.get('experiment', '')
```

Comment:
Any particular reason to have this default to empty instead of None?

Reply:
The experiment ID is required for any calls to the data provider API
(even though our current _implementation_ doesn’t strictly validate
this—after this PR, maybe it should), so it would have to be converted
from empty string to `None` at some point. It seemed cleanest to me to
reduce the space of input types as early as possible. No strong opinion.